src/components/structures/auth/Login.js

```diff
@@ -225,37 +223,6 @@ module.exports = React.createClass({
         }).done();
     },
 
-    _onLoginAsGuestClick: function(ev) {
-        ev.preventDefault();
-        ev.stopPropagation();
-
-        const self = this;
-        self.setState({
-            busy: true,
-            errorText: null,
-            loginIncorrect: false,
-        });
-
-        this._loginLogic.loginAsGuest().then(function(data) {
```

Comment:
It appears `loginAsGuest` is also now dead, so it should be removed as well I suppose?

Reply:
I'd rather not because of:
![image](https://user-images.githubusercontent.com/1190097/55643969-b92b2e00-5791-11e9-8cad-ce1d9eca4ba7.png)

Reply:
although the code is the same... so... goodbye dead code I guess